test/extended/util/certs/info/generate.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/origin/pkg/certs"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/image"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Fprintf(os.Stderr, "error: requires exactly one argument\n")
+		os.Exit(1)
+	}
+	outputDir := os.Args[1]
+
+	image.InitializeImages("")
+	exutil.WithCleanup(func() {})
+
+	var (
+		architectures = []string{"arm64", "amd64"}
+		platforms     = []string{"aws", "azure", "gcp", "openstack", "metal"}
+		networks      = []string{"ovn", "sdn"}
+	)
+	for _, architecture := range architectures {
```

Comment:
oh, is this the command producing a serialized version of https://github.com/openshift/library-go/blob/master/pkg/certs/cert-inspection/certgraphapi/type_registry.go#L4 ?

To do so, I think we need to read all the raw TLS info content (PR producing that updated here https://github.com/openshift/origin/pull/28321), then create a `certs[InClusterSecretLocation]=PKIRegistryCertKeyPairInfo` that will fail well on conflicting information for different combinations. And perhaps after our first version we grow the library-go API to indicate which topology, arch, platform, network it exists on.

Reply:
I've realized that we probably don't need multiple combinations. Certs don't change owner/description in different cluster topologies and in-cluster secrets can be a subset of expected secrets (to handle different secret names in older versions). 
So instead we'll have one `tls-artifacts` containing all known certs